mmv1/templates/terraform/examples/base_configs/iam_test_file.go.erb

```diff
@@ -261,6 +261,14 @@ func TestAcc<%= resource_name -%>IamBindingGenerated_withAndWithoutCondition(t *
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+<% end -%>
+<% if object.min_version.name == "ga" -%>
```

Comment:
I'm not sure why these import steps are all wrapped in conditions to make import only be tested in the `ga` version of tests. Should I remove them, or are the `beta` tests intentionally kept as a lighter version of `ga` tests?

Reply:
Usually the gate would be `unless` the version is GA to make sure the test is beta-only. I don't know of a reason that these tests should be GA-only.

Reply:
Cool, I'll stop these steps being ga-only and push!

Reply:
Actually, this might be best to save for a separate PR.

This logic appears to be linked to the lines below from [iam_test_file.go.erb](https://github.com/hashicorp/magic-modules/blob/f363d1c9fbfffc12a491a8dbde18dbc34bdbb77a/templates/terraform/examples/base_configs/iam_test_file.go.erb#L59-L63) and seems like it'd be confusing to lump that change into this PR?

```erb
<% unless object.min_version.name == "ga" -%>
		Providers:    testAccProvidersOiCS,
<% else -%>
		Providers:    testAccProviders,
<% end -%>
```

Reply:
Ah - I see.

setting `Providers` needs to be different depending on beta vs ga for the resource - so if checking the import state is only possible for `testAccProviders` then these are necessary, I guess. Seems like something to fix at some point in the future but +1 to leaving this all in place in this PR